code/modules/organs/limbs.dm

```diff
@@ -558,6 +558,17 @@ Note that amputating the affected organ does in fact remove the infection from t
 	if((limb_status & LIMB_ROBOT)) //Robotic limbs don't heal or get worse.
 		return
 
+	if(limb_status & LIMB_SPLINTED) //Splints gradually break over time.
+		if(splint_time <= 0)
+			remove_limb_flags(LIMB_SPLINTED)
+			to_chat(owner, span_userdanger("The splint on your [display_name] comes apart!"))
+		else
+			if(world.time - owner.last_move_time > 30)
+				return
+			splint_time = max(splint_time - 1, 0) //splint health decrements by one per lifetick, unless it's been more than 10 seconds since your last move.
```

Comment:
Could just reduce like `splint_time -= 1` and then add `splint_time = 0` where you remove the flag.